lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -254,8 +263,9 @@ bool UsdMaya_ReadJob::Read(std::vector<MDagPath>* addedDagPaths)
         std::make_pair(rootPathToRegister.GetString(), mMayaRootDagPath.node()));
 
     if (mArgs.useAsAnimationCache) {
-        MDGModifier dgMod;
-        MObject     usdStageNode = dgMod.createNode(UsdMayaStageNode::typeId, &status);
+        auto&        undoInfo = UsdUndoManager::instance().getUndoInfo();
```

Comment:
I'm a bit puzzled here.  Aren't we in the same method that has the undo muting guard?

Reply:
Ah yes, but the singleton and muting came after :) I'll revert this.